common/txmgr/txmgr.go

```diff
@@ -511,7 +518,13 @@ func (b *Txm[CHAIN_ID, HEAD, ADDR, TX_HASH, BLOCK_HASH, R, SEQ, FEE]) SendNative
 		Strategy:       NewSendEveryStrategy(),
 	}
 	etx, err = b.txStore.CreateTransaction(ctx, txRequest, chainID)
-	return etx, pkgerrors.Wrap(err, "SendNativeToken failed to insert tx")
+	if err != nil {
+		return etx, pkgerrors.Wrap(err, "SendNativeToken failed to insert tx")
```

Comment:
nit: can we use fmt.Errorf("msg: %w", err) here for error wrapping. the errors package is not supported any longer and the language now support for error wrapping